test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1

```diff
@@ -158,7 +182,7 @@
             $filePath | Should Exist
 
             $fileContent = Get-Content $filePath
-            $fileContent | Should Not Be $null
+            $fileContent | Should Not BeNullOrEmpty
```

Comment:
It seems it is unrelated the PR - please revert.

Reply:
can, but was cleaning up other instances of this, want me to pull it to a separate PR or?

https://github.com/PowerShell/PowerShell/pull/5815#discussion_r160291972

Reply:
Such changes increase the reviewing time because a reviewer must understand that this change is not related to code changes.

Thanks for clarify. I don't see the request. I am fine to leave the change.